### DISCUSSION TITLE
Sort by date modified

### DISCUSSION THREAD

**Discussion by RosySyn** [elementary/files#2215] [Ideas]
I'm having a good day, downloading various files to be used for a project later. I am caffeinated. _I am in the zone_.
I open up Files and navigate to my Downloads folder. I see 1,000 files of various sizes and types, a culmination of the week's hoarding. **I have attention deficit disorder.**
I right-click the pane, hoping to sort by Date Modified. _There is no such option_.
I weep, and manually comb through everything. This takes me about an hour to accomplish what I need to do.

Please. **I beg of you**. Add the option to sort by date modified. I can't even tell what I've downloaded _today_.

**Comment by jeremypw** [comment]
![Screenshot from 2023-06-13 10 35 17](https://github.com/elementary/files/assets/10513844/72e18026-657f-43cd-980a-f7241d916fa5)
![Screenshot from 2023-06-13 10 35 41](https://github.com/elementary/files/assets/10513844/954d8ee8-427e-49e1-a9aa-d58cfc979aeb)
You should be able to sort by date modified in Icon View (the folder context menu shows "Date" in the Sort submenu but it sorts on Date Modified.  For lots of files its probably easier to switch to List View and click on the "Modified" header to sort by data modified.
